static/src/stylesheets/layout/_header.scss

```diff
@@ -179,3 +147,648 @@ $c-guardian-services-action: #ffffff;
+}
+
+.new-header__nav-link {
+    @include fs-textSans(5);
+    @include trailing-slash-link;
+    line-height: $gs-baseline * 2.5;
+
+    // Override a from _lists.scss
+    color: $news-support-1;
+    cursor: pointer;
+
+    @include mq($from: mobile, $until: mobileLandscape) {
+        font-size: 16px;
+        font-size: 5.1vw;
+    }
+
+    @include mq($from: mobileMedium, $until: mobileLandscape) {
+        line-height: $gs-baseline * 3;
+    }
+
+    @include mq(mobileLandscape) {
+        font-size: 24px;
+        line-height: $gs-baseline * 3.5;
+    }
+
+    &:focus,
+    &:hover {
+        text-decoration: none;
+    }
+
+    .new-header__nav-link-item:first-child & {
+        padding-left: $gs-gutter / 2;
+    }
+
+    &:after {
+        @include trailing-slash;
+        color: $news-main-2;
+    }
+}
+
+/****************
+ * Menu Styling
+ ****************/
+
+.main-menu-container {
+    position: fixed;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    line-height: 1;
+    z-index: $zindex-main-menu;
+    box-shadow: 3px 0 16px rgba(0, 0, 0, .4);
+}
+
+.main-menu-container__overlay {
+    background-color: hsla(0, 0%, 0%, .5);
+    position: fixed;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    transition: opacity .2s cubic-bezier(.23, 1, .32, 1);
+    // Without this, in Safari/IOS this element appears above the menu
+    z-index: $zindex-overlay;
+    display: none;
+}
+
+.main-menu-container__menu {
+    height: 100%;
+    background-color: $guardian-brand-dark;
+    color: #ffffff;
+    font-family: $f-sans-serif-text;
+    text-transform: lowercase;
+    overflow: auto;
+    -webkit-overflow-scrolling: touch;
+    font-size: 20px;
+    will-change: transform;
+    transition: transform .2s cubic-bezier(.23, 1, .32, 1);
+
+    @include mq($until: mobileMedium) {
+        padding-top: $gs-baseline / 3;
+    }
+
+    @include mq(mobileMedium) {
+        //Larger font/icon size for larger devices
+        font-size: 22px;
+        padding-top: $gs-baseline;
+    }
+
+    .main-menu-container.off-screen & {
+        transform: translateX(-110%);
+        transition: transform .4s cubic-bezier(.23, 1, .32, 1);
+    }
+}
+
+.new-header__nav__button {
+    display: none;
+}
+
+.main-menu-container {
+    transform: translateX(-110%);
+    transition: transform .4s cubic-bezier(.23, 1, .32, 1);
+
+    @include mq($until: mobileMedium) {
+        margin-right: $gutter-small + ($veggie-burger-small / 2);
+    }
+
+    @include mq($from: mobileMedium, $until: mobileLandscape) {
+        margin-right: $gutter-medium + ($veggie-burger-medium / 2);
+    }
+
+    @include mq(mobileLandscape) {
+        margin-right: $gutter-large + ($veggie-burger-medium / 2);
+    }
+}
+
+.new-header__nav__button {
+    &:checked,
+    &[aria-expanded='true']  {
+        & ~ .main-menu-container {
+            transform: translateX(0%);
+            transition: transform .4s cubic-bezier(.23, 1, .32, 1);
```

Comment:
There it is again - maybe we could move this into a mixin which is scoped to the file ... What do you think?